aptos-move/aptos-debugger/src/lib.rs

```diff
@@ -154,13 +155,14 @@ impl AptosDebugger {
     where
         F: FnOnce(&mut SessionExt<StorageAdapter<DebuggerStateView>>) -> VMResult<()>,
     {
+        let mut features = Features::default();
+        features.update(FeatureFlag::TREAT_FRIEND_AS_PRIVATE, true);
```

Comment:
this feels like an anti-pattern -- we should get the features from the state view.. what if we just don't call update here and leave a todo instead?

Reply:
how about now?
```
        let state_view = DebuggerStateView::new(self.debugger.clone(), version);
        let state_view_storage = StorageAdapter::new(&state_view);
        let features = Features::fetch_config(&storage).unwrap_or_default();
        let move_vm = MoveVmExt::new(
            NativeGasParameters::zeros(),
            AbstractValueSizeGasParameters::zeros(),
            LATEST_GAS_FEATURE_VERSION,
            ChainId::test().id(),
            features,
        )
            .unwrap();
...
```